dashboard/app/views/home/_homepage.html.haml

```diff
@@ -60,3 +60,13 @@
     %a{href:'https://docs.google.com/a/code.org/document/d/1lHBthPKdKx3G-khyWYhVYQlfdi9Sa1L9EcOxEa-Xt14/edit'}
       %h3
         View Affiliate Playbook
+
+:javascript
+
+  $(document).ready(function() {
+    window.cleverTakeoverManager = new CleverTakeoverManager({
+      cleverLinkFlag: "#{session['clever_link_flag']}",
+      userIDToMerge: "#{session['clever_takeover_id']}",
+      mergeAuthToken: "#{session['clever_takeover_token']}",
+    });
+  });
```

Comment:
I think it would be preferable to move this `$(document).ready` handler into `_homepage.js` as well; and you wouldn't have to put `CleverTakeoverManager` on window anymore.  Pass the session variables here to the script via the `homepage_data` hash, above.